src/contour/ContourApp.cpp

```diff
@@ -178,6 +179,45 @@ auto withOutput(crispy::cli::flag_store const& flags, std::string const& name, C
     return callback(*out);
 }
 
+int ContourApp::documentationVT()
+{
+    using category = vtbackend::FunctionCategory;
+    using namespace std::string_view_literals;
+
+    std::string info;
+    auto back = std::back_inserter(info);
+    fmt::format_to(back, "#{}\n", "VT sequences");
+    fmt::format_to(back, "{}\n", "List of VT sequences supported by Contour Terminal Emulator.");
+    for (auto const& [category, headline]: { std::pair { category::C0, "Control Codes"sv },
+                                             std::pair { category::ESC, "Escape Sequences"sv },
+                                             std::pair { category::CSI, "Control Sequences"sv },
+                                             std::pair { category::OSC, "Operating System Commands"sv },
+                                             std::pair { category::DCS, "Device Control Sequences"sv } })
+    {
+
+        fmt::format_to(back, "## {}\n", headline);
+
+        for (auto const& fn: vtbackend::allFunctions())
+        {
+            if (fn.category != category)
+                continue;
+
+            // This could be much more improved in good looking and informationally.
+            // We can also print short/longer description, minimum required VT level,
+            // colored output for easier reading, and maybe more.
+            fmt::format_to(back,
+                           "`{:}` {:} [`{:}`] <br/>\n",
```

Comment:
You might avoid the use of the HTML-like `<br/>` tag by going with the table approach. I think what MS did there looks quite nice: https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences

This is possible for us, too. :)

Reply:
Update with tables